### PR TITLE
Updates for revamped billing page and usage alerts

### PR DESCRIPTION
**Pull request by jprcoliveira**
## When should this change go live?

- [x] This change is not yet live and should not be merged until Aug 1, 2023 (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by jprcoliveira** [comment]
@ultraschuppi I had some spare time so I started to draft some docs. No action needed just a heads up so there's no duplicated work ;)

**Comment by jprcoliveira** [comment]
@ultraschuppi this is ready for your review. Could you have a look today and see if all looks good to go?

File: docs/components/console/manage-organization/img/usage_alerts_logs.png
Comment (christinaausley) [inline docs/components/console/manage-organization/img/usage_alerts_logs.png:1]: This is nitpicky, but Organization is a bit cut off.

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit and one non-blocking comment. Nice work 👍

**Comment by christinaausley** [comment]
@jprcoliveira Are you ready to merge this in?

**Review by christinaausley — APPROVED** [review]

**Pull request merged**